JetStreamDriver.js

```diff
@@ -2235,6 +2235,257 @@ let BENCHMARKS = [
+            dllTextEncodingsWebUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Text.Encodings.Web.wasm",
+            dllTextJsonUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Text.Json.wasm",
+            dllAppUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/dotnet.wasm",
+        },
+        iterations: 20,
+        tags: ["Wasm", "dotnet"]
+    }),
+    new AsyncBenchmark({
+        name: "dotnet-aot-exceptions",
+        files: [
+            "./wasm/dotnet/exceptions.js",
+            "./wasm/dotnet/benchmark.js",
+        ],
+        preload: {
+            dotnetUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/dotnet.js",
+            dotnetNativeUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/dotnet.native.js",
+            dotnetRuntimeUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/dotnet.runtime.js",
+            wasmBinaryUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/dotnet.native.wasm",
+            icuCustomUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/icudt_CJK.dat",
+            dllCollectionsConcurrentUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Collections.Concurrent.wasm",
+            dllCollectionsUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Collections.wasm",
+            dllComponentModelPrimitivesUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.ComponentModel.Primitives.wasm",
+            dllComponentModelTypeConverterUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.ComponentModel.TypeConverter.wasm",
+            dllDrawingPrimitivesUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Drawing.Primitives.wasm",
+            dllDrawingUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Drawing.wasm",
+            dllIOPipelinesUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.IO.Pipelines.wasm",
+            dllLinqUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Linq.wasm",
+            dllMemoryUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Memory.wasm",
+            dllObjectModelUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.ObjectModel.wasm",
+            dllPrivateCorelibUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Private.CoreLib.wasm",
+            dllRuntimeInteropServicesJavaScriptUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Runtime.InteropServices.JavaScript.wasm",
+            dllTextEncodingsWebUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Text.Encodings.Web.wasm",
+            dllTextJsonUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/System.Text.Json.wasm",
+            dllAppUrl: "./wasm/dotnet/build-aot/wwwroot/_framework/dotnet.wasm",
+        },
+        iterations: 10,
+        tags: ["Wasm", "dotnet"]
+    }),
+    new AsyncBenchmark({
+        name: "dotnet-aot-raytracer",
+        files: [
+            "./wasm/dotnet/raytracer.js",
+            "./wasm/dotnet/benchmark.js",
+        ],
+        preload: {
```

Comment:
Could we deduplicate some of these preload files and use a common array?

Reply:
Yes, definitely. I wasn't sure what is the preference here. I'll put them to a common array